src/lib/mixpanel/mixPanelSingleton.ts

```diff
@@ -10,7 +8,7 @@ let _mixPanel: typeof Mixpanel | undefined = undefined
 
 // we need to be able to access this outside react
 export const getMixPanel = (): MixPanelType => {
-  const mixPanelEnabled = selectFeatureFlag(store.getState(), 'MixPanel')
+  const mixPanelEnabled = getConfig().REACT_APP_FEATURE_MIXPANEL
```

Comment:
means this won't work dynamically toggling flag at runtime, but avoids circular import hell